x-pack/plugin/search-business-rules/src/main/java/org/apache/lucene/search/CappedScoreQuery.java

```diff
@@ -33,6 +33,11 @@ public Query getQuery() {
         return query;
     }
 
+    @Override
+    public void visit(QueryVisitor visitor) {
+        query.visit(visitor);
```

Comment:
@romseygeek should it be:
```suggestion
        visitor.getSubVisitor(BooleanClause.Occur.MUST, this))
```
?

Reply:
Yes, that's a better option - this way a visitor can be informed of the existence of the CappedScoreQuery in the tree